test/autotest.py

```diff
@@ -606,12 +592,12 @@ def testKill():
 
   def testCheckpoint():
     #start checkpoint
-    coordinatorCmd(CKPT_CMD)
+    runDmtcpCommand(CKPT_CMD)
 
     #wait for files to appear and status to return to original
     # b'Kc' input to dmtcp_coordinator is equivalent to 'dmtcp_command -kc'
     WAITFOR(lambda: getNumCkptFiles(ckptDir)>0 and \
-                 (CKPT_CMD == b'Kc' or doesStatusSatisfy(getStatus(), status)),
+                 (CKPT_CMD == "Kc" or doesStatusSatisfy(getStatus(), status)),
```

Comment:
Same here. We don't need to break Python 2.7 for no special reason.
'Kc' and "Kc" are equivalent. But removing b is not equivalent.
Please do: b'Kc' -> b"Kc".